yt_dlp/extractor/olympics.py

```diff
@@ -0,0 +1,52 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import unified_strdate
+
+
+class OlympicsReplayIE(InfoExtractor):
+    _VALID_URL = r'(?:https?://)(?:www\.)?olympics\.com/tokyo-2020/(?:[a-z]{2}/)?replay/(?P<id>[^/#&?]+)'
+    _TESTS = [{
+        'url': 'https://olympics.com/tokyo-2020/en/replay/300622eb-abc0-43ea-b03b-c5f2d429ec7b/jumping-team-qualifier',
+        'info_dict': {
+            'id': '300622eb-abc0-43ea-b03b-c5f2d429ec7b',
+            'ext': 'mp4',
+            'title': 'Jumping Team Qualifier',
+            'release_date': '20210806',
+            'upload_date': '20210713',
+        },
+        'params': {
+            'format': 'bv',
+        },
+    }, {
+        'url': 'https://olympics.com/tokyo-2020/en/replay/bd242924-4b22-49a5-a846-f1d4c809250d/mens-bronze-medal-match-hun-esp',
+        'only_matching': True,
+    }]
+
+    def _real_extract(self, url):
+        id = self._match_id(url)
+        # The parameters are hardcoded in the webpage, it's not necessary to download the webpage just for these parameters.
+        # If in downloading webpage serves other functions aswell, then extract these parameters from it.
+        token_url = 'https://appovptok.ovpobs.tv/api/identity/app/token?api_key=OTk5NDcxOjpvY3N3LWFwaXVzZXI%3D&api_secret=ODY4ODM2MjE3ODMwYmVjNTAxMWZlMDJiMTYxZmY0MjFiMjMwMjllMjJmNDA1YWRiYzA5ODcxYTZjZTljZDkxOTo6NTM2NWIzNjRlMTM1ZmI2YWNjNmYzMGMzOGM3NzZhZTY%3D'
```

Comment:
This is generally not a robust approach, but given the nature of the website, I guess it's fair enough